README.md

```diff
@@ -6,21 +6,21 @@
 
 # MediBase3
 
-* **MediBase3** is a desktop application designed for doctors and medical professionals to manage patient contact details, appointments, medical conditions, allergies, and more.
+* **MediBase3** is a desktop application designed for doctors to manage patient contact details, appointments, medical conditions, **allergies**, and more.
   * Example usages:
     * A doctor can track patient information, medical history, and contact details efficiently.
-    * Medical professionals can schedule, update, and delete patient appointments.
-    * Easily manage patient medical conditions and track allergies to ensure safe treatment.
+    * Doctors can schedule, update, and delete patient appointments.
+    * Manages and tracks patients' medical conditions and **allergies** to enhance patient safety.
```

Comment:
Remove bold on allergies here, requested you to bold the one below not this.

Reply:
If you want to bold allergies, bold medical conditions as well, and other features. Else just remove.